solutions/usaco-1111.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: usaco-1111
+source: USACO Silver 2021 February
+title: Year of the Cow
+author: Melody Yu (Video)
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob2_silver_feb21.html)
+
+## Video Solution
+
+Note: The video solution might not be the same as other solutions. Code in C++.
```

Comment:
there are no other solutions right now, so is this necessary?

Reply:
what about the official solution?